src/containers/BooksForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createCreator } from '../actions/index';
+
+function mapDispatchToProps(dispatch) {
+  return {
+    addBook: bookObj => dispatch(createCreator(bookObj)),
+  };
+}
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: 'Category',
+    };
+    this.handleChanges = this.handleChanges.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChanges(event) {
+    if (event.target.id === 'title') {
+      this.setState({
+        title: event.target.value,
+      });
+    } else if (event.target.id === 'category') {
+      this.setState({
+        category: event.target.value,
+      });
+    }
+  }
```

Comment:
**[Optional] -** This way is totally correct, however, you could use `else` instead of `else if` and you could also use `name` instead of `id` and make use of array destructuring.